### PR TITLE
Adds command for visualization of post processing results.

### PR DESCRIPTION
**Pull request by tanushree04**
### Pull Request Description

- Adds command for visualizing scenario results after post processing. 
- Adds html file to the example_file folder to generate output html file of results.

### Checklist (Delete lines that don't apply)

- [x] Unit tests have been added or updated
- [ ] All ci tests pass (green)
- [x] An [ISSUE](https://github.com/urbanopt/urbanopt-cli/issues) has been created that this is addressing. Issues will get added to the Change Log when the change_log.rb script is run
- [x] This branch is up-to-date with develop

### CODE REVIEW

File: lib/uo_cli.rb [hunk 1]
```diff
@@ -459,6 +463,12 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
           scenario_report_features = reopt_post_processor.run_scenario_report_features(scenario_report: scenario_report, save_names_feature_reports: ['feature_optimization'] * scenario_report.feature_reports.length, save_name_scenario_report: 'feature_optimization')
           puts "\nDone\n"
         end
+      elsif @opthash.subopts[:visualize] == true
+        visualization = URBANopt::Scenario::ResultVisualization.create_visualization(@root_dir)
```
Comment (vtnate) [inline lib/uo_cli.rb:467]: Does this need to be set to a variable, or can `visualization = ` be removed? Katherine had a good suggestion about checking for tests passing locally, can you confirm that?

**Comment by tanushree04** [comment]
The test for visualization is failing because it requires the html file in the example file folder of the master branch https://github.com/urbanopt/urbanopt-cli/pull/116/files#diff-e4f12bf02f874628bd78d9cc86f25c58R468.

The reopt post processing scenario and feature tests also failed in the local run. https://github.com/urbanopt/urbanopt-cli/pull/116/files#diff-c0881479bcd379b2fb3fd8e1cba1198eR165, https://github.com/urbanopt/urbanopt-cli/pull/116/files#diff-c0881479bcd379b2fb3fd8e1cba1198eR165

File: lib/uo_cli.rb (hunk 1 above)
Comment (tanushree04) [reply lib/uo_cli.rb]: @vtnate the variable was not necessary, I removed it.

File: lib/uo_cli.rb [hunk 2]
```diff
@@ -459,6 +463,12 @@ def self.create_project_folder(dir_name, empty_folder = false, overwrite_project
           scenario_report_features = reopt_post_processor.run_scenario_report_features(scenario_report: scenario_report, save_names_feature_reports: ['feature_optimization'] * scenario_report.feature_reports.length, save_name_scenario_report: 'feature_optimization')
           puts "\nDone\n"
         end
+      elsif @opthash.subopts[:visualize] == true
+        URBANopt::Scenario::ResultVisualization.create_visualization(@root_dir)
+        html_in_path = "https://raw.githubusercontent.com/urbanopt/urbanopt-cli/master/example_files/scenario_comparison.html"
```
Comment (vtnate) [inline lib/uo_cli.rb:468]: LGTM

**Pull request merged**